migration/3.6.0/adminv2-convert-cross-roles.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import os
+import unicodedata
+from datetime import datetime
+import timeit
+from neo4jrestclient.client import GraphDatabase
+
+if len(sys.argv) >= 2:
+    print("Starting adminv2 cross roles conversion...")
+    print("Writing log to file: ./adminv2-convert-cross-roles.log...")
+    logFile = open("adminv2-convert-cross-roles.log", "w")
+    logFile.write("Adminv2 Cross Roles Conversion Log\n")
+    logFile.write("Date: " + str(datetime.now()) + "\n")
+    start = timeit.default_timer()
+    
+    neo4jHost = sys.argv[1]
+    preview = False
+    if len(sys.argv) == 3:
+        preview = True
```

Comment:
peut-être pour éviter une erreur bête, vérifier la valeur de l'argument preview, moi j'aurais surement utilisé le script comme ça:

```bash
./adminv2-convert-cross-roles.py myNeo4jHost true
```

puis:

```bash
./adminv2-convert-cross-roles.py myNeo4jHost false
```

Reply:
peut-être aussi printer si on est en preview ou pas, could be nice

Reply:
je vais garder l'argument "preview" mais vérifier la valeur ouai